src/main/java/com/redhat/devtools/lsp4ij/features/documentation/LSPDocumentationTarget.java

```diff
@@ -55,7 +56,13 @@ public TargetPresentation computePresentation() {
     @Nullable
     @Override
     public DocumentationResult computeDocumentation() {
-        return DocumentationResult.documentation(convertToHtml(contents, languageServer, file));
+        return DocumentationResult.documentation(getHtml());
+    }
+
+    @NotNull
+    @ApiStatus.Internal

```

Comment:
Need to be able to get this for `LSPSemanticTokenDocumentationProvider` for **Ctrl/Cmd+Mouseover** doc.